packages/@glimmer/runtime/lib/scanner.ts

```diff
@@ -83,14 +84,16 @@ export const ATTRS_BLOCK = '&attrs';
 export function layout(prelude: WireFormat.Statement[], head: WireFormat.Statement[], body: WireFormat.Statement[], symbolTable: ProgramSymbolTable) {
   let [, tag] = prelude.pop() as WireFormat.Statements.OpenElement;
   prelude.push([Ops.ClientSideStatement, ClientSide.Ops.OpenComponentElement, tag]);
+  prelude.push([Ops.ClientSideStatement, ClientSide.Ops.DidCreateElement]);
 
   let attrsSymbol = symbolTable.symbols.length + 1;
   symbolTable.symbols.push(ATTRS_BLOCK);
 
   let statements = prelude
     .concat([[Ops.Yield, attrsSymbol, EMPTY_ARRAY]])
     .concat(head)
-    .concat(body);
+    .concat(body)
+    .concat([[Ops.ClientSideStatement, ClientSide.Ops.DidRenderLayout]]);
```

Comment:
I tried it quickly but it makes TS sad, feel free to follow up (this thing will probably be removed soon anyway)